Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -45,11 +45,13 @@
 #endif
 
 NSString * const NSReadAccessURLDocumentOption = @"ReadAccessURL";
+NSString * const WKReadAccessFileURLsOption = @"_WKReadAccessFileURLsOption";
```

Comment:
I feel like the constant name should also be prefixed by an underscore (`_WKReadAccessFileURLsOption`) since it's not API yet (and to match the value), but I'm not sure what the convention is.

I also see a style like this used in some places:
```
NSString * const _WKReadAccessFileURLsOption = @"WKReadAccessFileURLsOption";
```

The latter style documents that the variable is not public API, but uses (what would be) a public API key name in case it becomes public later.

Please check with someone who does regular API review to find out the correct way to define these before landing.

Reply:
Since this is not a persistent key (it's not stored in User Defaults, for example), I think it's safe for us to use the underscored version of the string until an explicit decision to make it API.

I'll revise the name of the constant to use a prefix, too, to signify that these are currently SPI. I'll also file a radar to make it API for the next SDK update.